virttest/block.py

```diff
@@ -0,0 +1,20 @@
+"""
+Utility functions to handle block related actions.
+
+"""
+
+from . import qemu_monitor
+
+
+def get_block_virtual_size(dev_name):
+    """
+    Get block virtual-size of specified block device with dev_name.
+
+    :param dev_name: block device name, like "drive-hotadd" etc.
+    :return: Matched block device virual size.
```

Comment:
Please add measurement unit of the size, eg. M, G

Reply:
ok, thanks, will update